hudi-cli/src/main/java/org/apache/hudi/cli/commands/RepairsCommand.java

```diff
@@ -71,10 +72,8 @@ public class RepairsCommand {
   @ShellMethod(key = "repair deduplicate",
       value = "De-duplicate a partition path contains duplicates & produce repaired files to replace with")
   public String deduplicate(
-      @ShellOption(value = {"--duplicatedPartitionPath"}, defaultValue = "", help = "Partition Path containing the duplicates")
-      final String duplicatedPartitionPath,
-      @ShellOption(value = {"--repairedOutputPath"}, help = "Location to place the repaired files")
-      final String repairedOutputPath,
+      @ShellOption(value = {"--duplicatedPartitionPath"}, defaultValue = "", help = "Partition Path containing the duplicates") final String duplicatedPartitionPath,
+      @ShellOption(value = {"--repairedOutputPath"}, help = "Location to place the repaired files") final String repairedOutputPath,
```

Comment:
Not sure why we do these format changes

Reply:
> Not sure why we do these format changes

I used the shortcut keys in IntelliJ IDEA to format it.